src/resources/arranger_es_metadata.json

```diff
@@ -1190,7 +1190,7 @@
             "displayValues": {},
             "quickSearchEnabled": false,
             "field": "file.index_file.object_id",
-            "displayName": "File Index File Object Id",
+            "displayName": "Object ID",
```

Comment:
sorry missed this before

consistency with above fields:
Index File Object ID

Reply:
did `Index File Size` too :)